tests/test_consumer/test_extractors/test_extractors_dataset.py

```diff
@@ -35,8 +35,6 @@ def test_extractors_extract_dataset_hdfs():
         ),
         name="/user/hive/warehouse/mydb.db/mytable",
     )
-    assert not extract_dataset_aliases(dataset)
-    assert not extract_dataset_symlinks(dataset)
```

Comment:
Instead of removing assert, replace it with `extract_dataset_with_symlinks` call, and check that there are no symlinks

Reply:
Yeah, also thinking about adding test for `extraxtors_interactions`